asset-transfer-basic/chaincode-go/chaincode/smartcontract.go

```diff
@@ -0,0 +1,193 @@
+package chaincode
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+)
+
+// SmartContract provides functions for managing an Asset
+type SmartContract struct {
+	contractapi.Contract
+}
+
+// Asset describes basic details of what makes up a simple asset
+type Asset struct {
+	ID             string `json:"ID"`
+	Color          string `json:"color"`
+	Size           int    `json:"size"`
+	Owner          string `json:"owner"`
+	AppraisedValue int    `json:"appraisedValue"`
```

Comment:
suggest to follow snake_case for json keys. ``json:"appraised_value"``

Reply:
The example is generally idiomatic of the fabric standard core configuration. Go actually specifies that naming conventions should follow the conventions of the implementation where possible, and absent that they should at least follow a consistent convention. As JSON also specifies no standard convention, the fabric convention is used.